libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -0,0 +1,56 @@
+mutation UpdateClientProfile($data: UpdateClientProfileInput!) {
+  updateClientProfile(data: $data) {
+    ... on OperationInfo {
+      messages {
+        kind
+        field
+        message
+      }
+    }
+
+    ... on ClientProfileType {
+      id
+    }
+  }
+}
+
+query GetClientProfile($id: ID!) {
+  clientProfile(pk: $id) {
+    ... on ClientProfileType {
+      id
+      address
+      dateOfBirth
+      gender
+      hmisId
+      nickname
+      phoneNumber
+      preferredLanguage
+      pronouns
+      spokenLanguages
```

Comment:
maybe we can remove this since we're not using this field yet

Reply:
done